lib/matplotlib/projections/polar.py

```diff
@@ -591,6 +605,14 @@ def format_coord(self, theta, r):
         Return a format string formatting the coordinate using Unicode
         characters.
         """
+        # Strip away any units
+        r = self.convert_xunits(r)
+        if not isinstance( theta, float ):
+            theta = self.convert_yunits(theta)
+            # Stripping away units makes theta be in radians.
+            # convert to degrees
+            theta = theta * (180.0 / np.pi)
```

Comment:
I think you're right. Even though this function calls the arguments `theta`/`r`, it's called by matplotlib, so it's (sorta) `x`/`y`, meaning "`theta`" is in radians.